work_items/kyc-vcs/requirements-to-accept-a-kyc-vc.md

```diff
@@ -0,0 +1,590 @@
+
+6. Companies also store a copy of the photo ID and/or the personal data from the ID, through OCR technology, at field level. This typically includes: Full name, Date of Birth, ID Type, ID Number, ID Country and address (if applicable). Therefore, we may want to consider including the personal information on the identity document within the verifiable credential evidence. 
+
+To obtain the evidence that the KYC Verifiable Credential was verified, the Financial Institution (or Verifier) will specify the Verification Evidence credential schemas and types they accept via a [Presentation Request](https://identity.foundation/presentation-exchange/#presentation-request).
+
+
+Such KYC Evidence may be associated with KYC Verifiable Credentilas **OR** provided as separate credentials. Such a credential, using the [Verifiable Credential Data Model Evidence Property](https://www.w3.org/TR/vc-data-model/#evidence) may appear as follows:
+
+**<span style="text-decoration:underline;">Example 4: KYC VC Evidence </span>**
+
+```json
+{
+   "@context": ["https://www.w3.org/2018/credentials/v1"],
+   "id": "8beaf6db-c636-4874-9997-445b31ea2596",
+   "type": ["VerifiableCredential", "KYC-Evidence"],
+   "issuer": "did:example:KYC-Issuer",
+   "credentialSubject": {
+      "id": "did:example:1a2b3c4d5e6f7g",
+      "name": {
+         "familyName": "Doe",
+         "givenName": "Jane"
+      },
+      "birthDate": "1999/12/25",
+      "postalAddress": {
+         "streetAddress": "31 South Main Street",
+         "country": "USA",
+         "locality": "Anytown",
+         "region": "Anystate",
+         "postalCode": "12345"
+      },
+      "taxId": "123-45-6789"
+   },
+   "evidence": [{
+      "id": "https://kyc-provider.com/evidence/evidence-id.json",
+      "type": ["KYC-Documentary-Evidence"],
+      "method": "Documentary",
+      "source": "Photo ID",
+      "match": {
+         "result": "Partial match",
+         "unmatchedData": "SSN"
+      },
+      "verifierName": "Jumio",
+      "documentType": "Passport",
+      "issuingCountry": "USA",
+      "idNumber": "123456789",
+      "expiryDate": "2026/12/25"
+   }],
+   "proof": { "..." }
+}
+
+```
+
+### 4. Address Verification Evidence
+
+Address verification is a regulatory requirement to onboard a customer in certain jurisdictions including in the UK, Europe and parts of Asia. This same type of credential can also be used to demonstrate proof of “Purpose Of Account”, if a bank or other financial account statement is used to validate a physical address.
+
+The Address or Account Verification Evidence will be requested by a Financial Institution when:
+
+
+1. Regulatory requirements dictate that address verification is required to onboard a customer based on risk; or
+2. Enhanced Due Diligence is performed (due to the risk associated with an account or customer) and proof of the purpose account or source of funds is required (either via a regulatory requirement or internal policy).
+
+<table>
```

Comment:
Just realized I need to add Account Status to the table and the example. Here is the description to include in the table: The account status is important to determine if an account is valid and can be used. The account status types can include: active, blocked, inactive, bank account frozen, non-payment bank account, etc. There are standard ACH return codes which can also be used here (these are typical responses to transaction requests) but would not be applicable for open/active accounts. Also, given the different account types possible, mapping to standardized responses would be beneficial.

Reply:
added